src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java

```diff
@@ -24,7 +24,21 @@
      * @return The embed builder for call chaining.
      */
     public static EmbedBuilder insertResponseTracker(EmbedBuilder builder, long trackedMessageId) {
-        return builder.setFooter("re: " + trackedMessageId);
+        return  insertResponseTracker(builder, trackedMessageId,
+                "If you delete your invocation message, this will be deleted. 🥄");
```

Comment:
I would not split this into two methods, if always only the one is used.
And why the spoon? This is not spoon-feeding.
I'd write "If you delete your invocation message, this response will be deleted"